### PR TITLE
Let's finally make batch legible

### PR DESCRIPTION
**Pull request by anton-br**
This pull request is aimed to restructuring batch actions. There are no changes in actions, just position in batch have been changed. I Suggest to split actions as follows:

__Decorators & Support functions:__
def _init_component
def _post_filter_by_mask
def copy_meta

__Load and Dump__
def load
def _load_segy
def _load_from_segy_file
def _load_from_index
def dump
def _dump_segy
def _dump_split_segy
def _dump_single_segy
def _dump_picking
def _dump_geometry_flags

__Crops Actions__
def make_grid_for_crops
def crop
def __crop
def assemble_crops

__Standardize Actions__
def standardize
def equalize

__Useful Actions__ (In this section we have frequently used actions with different purposes. I couldn't find a separate group for them, so I combined them into one.)
def correct_spherical_divergence
def _correct_sph_div
def drop_zero_traces
def hodograph_straightening
def sort_traces
def _sort

__Picking Actions__
def picking_to_mask
def mask_to_pick
def shift_pick_phase

__Plotters__
def seismic_plot
def crops_plot
def gain_plot
def spectrum_plot
def statistics_plot

__Rarely Used Actions and Functions__(Here we have functions that whether haven't used at all or uses only in one notebook)
def apply_along_axis
def band_pass_filter
def energy_to_picking
def items_viewer
def mcm
def pad_traces
def slice_traces
def to_2d
def trace_headers

Look forward to hearing your thoughts!

### CODE REVIEW

**Comment by a-arefina** [comment]
I would place Useful Actions to the end near Rarely Used Actions and Functions and join them into group named Misc, because "rare" and "often" can change sometimes and doesn't actually define those groups

**Comment by anton-br** [comment]
> I would place Useful Actions to the end near Rarely Used Actions and Functions and join them into group named Misc, because "rare" and "often" can change sometimes and doesn't actually define those groups

I don't want to join them into one group because functions from Rarely used actions are not used in our projects. Also, i don't want to delete them because one of us may need them later (in this case we move this function to "useful functions").

**Comment by dpodvyaznikov** [comment]
'useful' and 'rarely used' are bad structure elements.
When I add a new action, how do I know that it is going to be used often? And if I have a very useful, yet rarely used action, where do I put it?

**Comment by nikita-klsh** [comment]
I would suggest the following structure:
1. Load actions.
2. Actions that processing any component in any way(almost all our actions) sorted alphabetically.
3. Dump actions
4. Plot actions

Or to just sort everything alphabetically

**Comment by anton-br** [comment]
Based on your comments, the resulted structure is following:

__Decorators & Support functions:__

def _init_component
def _post_filter_by_mask
def copy_meta
def items_viewer

__Load and Dump__

def load
def _load_segy
def _load_from_segy_file
def _load_from_index
def dump
def _dump_segy
def _dump_split_segy
def _dump_single_segy
def _dump_picking
def _dump_geometry_flags

__Data Process Actions__
__DPA. Cropping Actions__

def make_grid_for_crops
def crop
def __crop
def assemble_crops

__DPA. Standardize Actions__

def standardize
def equalize

__DPA. Misc__

def apply_along_axis
def band_pass_filter
def correct_spherical_divergence
def _correct_sph_div
def drop_zero_traces
def hodograph_straightening
def mcm
def pad_traces
def slice_traces
def sort_traces
def _sort
def to_2d
def trace_headers

__DPA. Picking Actions__

def picking_to_mask
def mask_to_pick
def shift_pick_phase
def energy_to_picking

__Plotters__

def seismic_plot
def crops_plot
def gain_plot
def spectrum_plot
def statistics_plot

File: seismicpro/src/seismic_batch.py [hunk 1]
```diff
@@ -576,246 +565,450 @@ def _dump_geometry_flags(self, src, path, columns=('FieldRecord',)):
...
+            crop_counts[x:x1, y:y1] += 1
+
+        crop_counts[crop_counts == 0] = 1
+
+        getattr(self, dst)[pos] = res / crop_counts
         return self
 
+    #---------------------------------------
+    # DPA. Standardize Actions
+    #---------------------------------------
```
Comment (nikita-klsh) [inline seismicpro/src/seismic_batch.py:763]: How about making `#------------` lines longer, like 80 or 120 symbols? So they distinguish blocks really well.
Also i believe its `Normalize actions`. `Standardize and equalize` are particular cases.

**Review by a-arefina — APPROVED** [review]

**Pull request merged**